### PR TITLE
Created Italian translation

### PR DESCRIPTION
**Pull request by casungo**

### CODE REVIEW

File: lang/Italiano.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            "getting_started": "Per iniziare",
+            "getting_started_text": "1. Effettua il login nell'applicazione.\n2. Assicurati che il tuo account Twitch sia collegato a tutte le campagne per cui sei interessato a ottenere i drop.\n3. Se sei interessato a ottenere tutto, deseleziona \"Solo priorità\" e premi su \"Ricarica\".\n4. Se vuoi ottenere prima giochi specifici, usa la lista \"Priorità\" per impostare una lista ordinata di giochi a tua scelta. I giochi in cima alla lista verranno ottenuti prima.\n5. Mantieni l'opzione \"Solo priorità\" selezionata, per evitare di ottenere giochi che non sono nella lista delle priorità. Oppure non farlo - dipende da te.\n6. Usa la lista \"Escludi\" per dire all'applicazione quali giochi non devono mai essere ottenuti.\n7. Cambiare il contenuto di una delle liste, o cambiare lo stato dell'opzione \"Solo priorità\", richiede di premere su \"Ricarica\" perché le modifiche abbiano effetto."
+        }
+    }
+}
```
Comment (DevilXD) [inline lang/Italiano.json:161]:
```suggestion
}

```

Comment (DevilXD) [reply lang/Italiano.json:161]: The file has to end with a single, empty new line, per the suggestion.

**Review by DevilXD — CHANGES_REQUESTED** [review]
Hello o/

Overall, it looks okay. Please add a credit line to the README file as a part of this PR too, if possible.

**Comment by casungo** [comment]
Sure, commit coming right up :)

**Comment by DevilXD** [comment]
Perfect! Thank you for the contribution 🙂

**Pull request merged**